tests/acceptance/blocks/thememap.js

```diff
@@ -29,6 +38,46 @@ class ThemeMap {
   async clickSearchThisAreaButton () {
     await t.click(this._searchThisAreaButton);
   }
+
+  async clickMap () {
+    await t.click(this._canvas);
+  }
+
+  async getZoom () {
+    const zoom = await ClientFunction(() => {
+      return ANSWERS
+        .components
+        ._activeComponents
+        .find(component => component.name === 'VerticalFullPageMapOrchestrator')
+        .currentZoom
+    })();
+    return zoom;
+  }
+
+  /**
+   * Waits for the map zoom to sabilize
+   */
+  async waitForZoomStabilization () {
+    const pollingIntervalMsecs = 500;
+    const minNumStableIntervals = 3;
+    let previousZoom = await this.getZoom();
+    let numStableIntervals = 0;
+    let isZoomStabilized = false;
+
+    while (!isZoomStabilized) {
```

Comment:
Do we need a timeout here? I'm wondering if there's a situation in which we could get stuck waiting indefinitely.

Reply:
I don't think we need one. The zoom value is actually very stable. But I can add a `maxNumPolls` just to be safe